packages/server-wallet/src/db/migrations/20200915140300_chain_service.ts

```diff
@@ -0,0 +1,19 @@
+import * as Knex from 'knex';
+
+const channels = 'channels';
+const chainServiceRequests = 'chain_service_requests';
+const fundingStrategy = 'funding_strategy';
+
+export async function up(knex: Knex): Promise<any> {
+  await knex.schema.alterTable(channels, table => {
+    table.specificType(chainServiceRequests, 'text[]').notNullable();
```

Comment:
Should we add some DB constraints on the values of these columns?

Reply:
Any specific ones you are thinking about? Maybe restrict the set of strings that can be stored?

Reply:
Yeah.

I personally prefer DB constraints, though it has the downside of "now, your validation can live in two places".